tests/test_fio_workload.py

```diff
@@ -16,12 +17,43 @@
 logger = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope='class')
+def test_fixture(request):
+    """
+    Fixture only for teardown
+    """
+
+    self = request.node.cls
+
+    def finalizer():
+        teardown(self)
+    request.addfinalizer(finalizer)
+    setup(self)
+
+
+def setup(self):
```

Comment:
no need for setup if doesn't do anything.
In the fixture just remove the call to setup

Reply:
Done